client/state/notices/middleware.js

```diff
@@ -91,7 +82,8 @@ export function onPostSaveSuccess( dispatch, action ) {
  */
 
 export const handlers = {
-	[ ACCOUNT_RECOVERY_FETCH_FAILED ]: dispatchError( translate( 'An error occurred while fetching for your account recovery settings.' ) ),
+	...accountRecoveryNoticeHandlers,
+
```

Comment:
Oh, and 4b1373f29f4deda9732c8da7421cdf057aac1926. I forgot to remove the import statement in the previous commit :)